src/gerbil/runtime/c3.ss

```diff
@@ -19,13 +19,21 @@
 ;;;
 ;;; C3 has since been adopted for multiple inheritance by many modern
 ;;; object systems for its ordering consistency properties:
-;;; OpenDylan, Python 2.3, Raku, Parrot, Scala, Solidity, PGF/TikZ.
+;;; OpenDylan, Python 2.3, Raku, Parrot, Solidity, PGF/TikZ.
 ;;;       https://en.wikipedia.org/wiki/C3_linearization
 ;;;       https://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.19.3910
 ;;;
-;;; We dub C4 our extension with the 4th constraints for single-inheritance.
+;;; We dub C4 our extension with the 4th constraint for single-inheritance.
 ;;;
 ;;; See tests in ../test/c3-test.ss
+;;;
+;;; PS: Common Lisp, and the earlier tradition of multiple inheritance
+;;; (all the way back to Flavors), that we follow here, calls "classes"
+;;; the things with multiple inheritance and "structs" the things with
+;;; single inheritance only. Smalltalk and after it Java, and the earlier
+;;; and more prevalent tradition of single inheritance, calls "classes"
+;;; the things with single inheritance and "traits" the things with multiple
+;;; inheritance (after Mesa). Wonderful nomenclature, right?
```

Comment:
Love it :) Mentioning "Flavors" brought a warm fuzzy feeling and made this old man feel young again.